platform/android/export/export.cpp

```diff
@@ -1333,12 +1333,28 @@ class EditorExportAndroid : public EditorExportPlatform {
 
 	virtual bool can_export(const Ref<EditorExportPreset> &p_preset, String &r_error, bool &r_missing_templates) const {
 
+		String err;
 		r_missing_templates = find_export_template("android_debug.apk") == String() || find_export_template("android_release.apk") == String();
 
+		if (p_preset->get("custom_package/debug") != "") {
+			if (FileAccess::exists(p_preset->get("custom_package/debug"))) {
+				r_missing_templates = false;
```

Comment:
The problem with this approach is that if only one custom package is defined, then we consider that it's possible to export - but if it's say the debug package, and you want to export a release binary with no corresponding template, then it will error out.

That being said, the `EditorExportPlatformPC` uses a similar logic, so I guess it's OK to use the same for Android and JS. This whole system really needs a refactor though, but that's out of scope of this issue.